Objects/Objects/BuiltElements/Room.cs

```diff
@@ -28,26 +28,26 @@ public Room(string name, string number, Level level, [SchemaMainParam] Point bas
     /// <summary>
     /// SchemaBuilder constructor for a Room
     /// </summary>
-    /// <remarks>Assign units when using this constructor due to <paramref name="height"/> param</remarks>
+    /// <remarks>Assign units when using this constructor due to <propref name="height"/> prop</remarks>
```

Comment:
Is `propref` valid tag? Rider doesn't recognise it. I've been using `<see cref="height"/>` to reference properties